packs/smart_items/assets/rising_pillar_temple/composite.json

```diff
@@ -28,7 +28,11 @@
       "data": {
         "0": {
           "json": {
-            "componentIds": [1102, 2709697014, 1020]
+            "componentIds": [
```

Comment:
the sync components should be in the format:
```
{
  "name": "core-schema::Sync-Components",
  "data": {
    "0": {
      "json": {
        "value": [
          "core::Animator",
          "core::AudioSource",
          "asset-packs::States"
        ]
      }
    }
  }
},
```

Reply:
hmmm ok
this is how the json gets auto-generated when creating a Custom Item
I'll change it, but this step adds friction to creating smart items in general, would be nice if in the future we don't have to do this